enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -626,91 +626,72 @@ OSSM 2.4.
 
 #### Gateway Topology
 
-Users have the option to deploy their Gateways using two distinct topologies: shared gateways or dedicated gateways.
-Each topology addresses specific customer requirements, comes with different security implications (see [Security Policy](#security-policy)),
-and will be supported in OpenShift.
+A Kubernetes Gateway resource represents a proxy server (e.g., [Envoy]) and
+manages its life-cycle. Once deployed, it is programmed with route resources
+(e.g., HTTPRoute, GRPCRoute – collectively, xRoutes) to handle application
+traffic.
 
-##### Shared Gateway Topology
+There are two deployment options:
+
+* Dedicated Gateways
+* Shared Gateways (not yet supported)
+
+Which represent different ingress topologies. We'll go into more details about
+each of these in the following sections.
+
+[Envoy]:https://github.com/envoyproxy/envoy
+
+##### Dedicated Gateway Topology
 
-With shared gateway topology, a single load balancer type service and proxy deployment is created and serves
-routes across multiple namespaces. The Gateway must allow the desired application namespaces via the
-`spec.listeners.allowedRoutes.namespaces` field. This topology mirrors the existing OpenShift router topology and
-is ideal for users that are cost sensitive and do not have strict performance or security requirements.
+In a dedicated Gateway topology, a load balancer Service and a Deployment
+for the proxy server are deployed within a dedicated namespace to serve the
+xRoutes. This setup is ideal for multi-tenant environments or for applications
+that need a dedicated gateway to satisfy specific performance requirements.
 
 ```mermaid
 flowchart TD
     subgraph foo-app namespace
         xRoute
-    end
-    subgraph bar-app namespace
-        xRoute2
-    end
-    subgraph openshift-ingress namespace
         Gateway
         ProxyDeployment
         LBService
     end
-    Gateway[Gateway] -.-> ProxyDeployment(Proxy Deployment)
-    Gateway[Gateway] -.-> LBService(LB Service)
     xRoute[xRoute] --> Gateway[Gateway]
-    xRoute2[xRoute] --> Gateway[Gateway]
 ```
 
-This cross-namespace topology can only be implemented using the Gateway-to-xRoute hierarchy, as Istio does not support
-cross-namespace Gateway merging. Therefore, a limitation of this topology is that xRoutes lack an API for
-specifying TLS certificates, requiring all application certificates to be managed within a cluster-wide Gateway.
+##### Shared Gateway Topology
 
-Users can have multiple shared Gateways, similar to the concept of [sharding](https://docs.openshift.com/container-platform/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html#nw-ingress-sharding_configuring-ingress-cluster-traffic-ingress-controller) in OpenShift. 
+> **Warning**: Shared Gateways are not currently supported, as the upstream
```

Comment:
The concept of shared Gateways is not the same as the concept of merging Gateway listeners.

Reply:
deconflated, and note specifically calls out merging now :+1: